partcad/src/partcad/builtin/render/render_svg.py

```
@@ -94,36 +113,15 @@ def process(path, request):
-            line_weight=request["line_weight"],
+            line_weight=request.get("line_weight", 1.0),
         )
-        # exporter.add_layer(
-        #     "Hidden",
-        #     line_color=(32, 64, 32),
-        #     line_type=b3d.LineType.ISO_DOT,
-        # )
         try:
             exporter.add_shape(visible, layer="Visible")
-            # exporter.add_shape(hidden, layer="Hidden")
         except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)